<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Оцифровщик </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Подключение</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="connect_pushButton">
         <property name="text">
          <string>Подключится</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ip_lineEdit">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>192.168.1.10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="noRealTime_groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Режим No realtime</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Размер, кБ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="size_spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>64</number>
           </property>
           <property name="maximum">
            <number>31744</number>
           </property>
           <property name="singleStep">
            <number>64</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="start_pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Запуск приема</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Принято байт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="receiveSize_lcdNumber">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_pushButton">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="realTime_groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Режим Real-Time</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="realTime_pushButton">
         <property name="text">
          <string>Старт</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="framesNum_label">
           <property name="text">
            <string>Принято кадров</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="rtFrames_lcdNumber">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="test_checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Режим ТЕСТ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adcSpi_pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Запись в АЦП</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clkdistSpi_pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Запись CLKDIST</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ioExp_pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Запись в I/O Expander</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="pwm_groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Управление ШИМ</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="pwm_checkBox">
         <property name="text">
          <string>Включение </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Частота, Гц</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="pwmFreq_spinBox">
         <property name="showGroupSeparator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>10000</number>
         </property>
         <property name="maximum">
          <number>15000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Коэф. заполнения, %</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="pwmDC_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pwm_pushButton">
         <property name="text">
          <string>Установить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="dds_groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Управление DDS</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Частота, Гц</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="ddsFreq_spinBox">
         <property name="showGroupSeparator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200000000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Амплитуда, отсчеты ЦАП</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="ddsAmp_spinBox">
         <property name="maximum">
          <number>4095</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="dds_pushButton">
         <property name="text">
          <string>Установить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>131</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="save_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="save_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранть данные в файл</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
